src/layouts/MultiDhoLayout.vue

```diff
@@ -315,7 +379,145 @@ export default {
+          description = I18n.t('notifications.proposalHasPassed', { accountname: notification.user })
+          break
+        case ('proposals.rejected'):
+          icon = require('~/assets/icons/notifications/proposal-rejected.png')
+          title = I18n.t('notifications.proposalRejected')
+          description = I18n.t('notifications.proposalHasntPassed', { accountname: notification.user })
+          break
+        case ('system.claimable_period'):
+          icon = require('~/assets/icons/notifications/claimable-period.png')
+          title = I18n.t('notifications.claimablePeriod')
+          description = I18n.t('notifications.youHaveClaimablePeriod', { value: notification.content?.periods })
+          break
+        case ('assignment.extension_expiry'):
+          icon = require('~/assets/icons/notifications/extend-assignment.png')
+          title = I18n.t('notifications.extendYourAssignment')
+          description = I18n.t('notifications.youStillHave', { days: notification.content?.days })
+          break
+        case ('assignment.approved'):
+          icon = require('~/assets/icons/notifications/assignment-approved.png')
+          title = I18n.t('notifications.assignmentApproved')
+          description = I18n.t('notifications.yourAssignmentHasBeenApproved')
+          break
+        case ('assignment.rejected'):
+          icon = require('~/assets/icons/notifications/assignment-rejected.png')
+          title = I18n.t('notifications.assignmentRejected')
+          description = I18n.t('notifications.yourAssignmentHasntBeenApproved')
+      }
+      return {
+        icon: icon,
+        title: title,
+        description: description,
+        createdDate: createdDate
+      }
+    },
+    initNotifications () {
+      if (!localStorage.getItem('notifications')) {
+        const notifications = []
+        this.notifications.forEach((notification) => {
+          notifications.push({
+            ...notification,
+            showed: false
+          })
+        })
+        this.localNotifications = notifications
+        localStorage.setItem('notifications', JSON.stringify(notifications))
+      } else {
+        const parsedNotifications = JSON.parse(localStorage.getItem('notifications'))
+        this.notifications.forEach((notification) => {
+          const isMatched = parsedNotifications.find(item => item.id === notification.id)
+          if (!isMatched) {
+            parsedNotifications.push({
+              ...notification,
+              showed: false
+            })
+          }
+        })
+        this.localNotifications = parsedNotifications
+        localStorage.setItem('notifications', JSON.stringify(parsedNotifications))
+      }
+    },
+    countObjectsWithKeyValue(array, key, value) {
+      let count = 0
+      array.forEach(obj => {
+        if (obj[key] === value) {
+          count++
+        }
+      })
+      return count
+    },
+    readNotification (id) {
```

Comment:
@evgenibir readNotification should call GraphQL mutation to update `read` property to true